dotCMS/src/main/java/com/dotmarketing/portlets/director/action/DirectorAction.java

```diff
@@ -87,6 +88,29 @@ protected IHTMLPage loadPage(String inode, User user) throws DotDataException, D
 	    return APILocator.getHTMLPageAssetAPI().fromContentlet(APILocator.getContentletAPI().find(inode, user, false));
 	    
 	}
+	
+	/**
+     * Invalidates Cached Resources of Page
+     * in order to refresh any recent update on
+     * contents that were added/removed from it
+     * 
+     * @param htmlPage
+     *            - The Legacy or Content Page that has changed.
+     * @param user
+     *            - The user performing this action.
+     * @throws DotStateException
+     * @throws DotDataException
+     *             An error occurred when persisting the changes in the
+     *             database.
+     * @throws DotSecurityException 
+    */
+    protected void refreshPage(IHTMLPage htmlPage, User user) 
+            throws DotStateException, DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)